src/Interfaces/Options/Particles/Shape/IShapeValues.ts

```diff
@@ -0,0 +1,4 @@
+export interface IShapeValues {
+    fill?: boolean;
+    close?: boolean;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)